lang/dart.md

```diff
@@ -0,0 +1,188 @@
+---
+lang: Dart
+syntax: Dart
+title:  Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+## What is Dart?
+Dart is a new general and open source programming language with a vibrant community developed by Google Inc. and its official website is https://dart.dev. It was first announced as a public preview on October 10, 2011; it has now reached version 1.10. World class language designers and developers are involved in this project, namely, Lars Bak and Kasper Lund (known for their V8 JavaScript engine embedded in the Chrome browser, which revolutionized performance in the JavaScript world), and Gilad Bracha (a language theorist known for the development of the Strongtalk and Newspeak languages and for the Java specification). Judging by the huge amount of resources and the number of teams working on it, it is clear that Google is very serious about making Dart a success.
+
+## What is Dart used for?
+Dart programming language can be used for web and mobile applications and is also suitable for server and desktop applications.
+
+Dart wasn’t popular and was known as an orphan, but it had a compiler for a Dart VM, also Google had all the control about this language.
```

Comment:
```suggestion
The Dart programming language can be used for web and mobile applications and is also suitable for server and desktop applications.</br>
Dart wasn’t popular and was known as an orphan, but it had a compiler for a Dart VM, also Google had all the control about this language.
```

This will make it look like

The Dart programming language can be used for web and mobile applications and is also suitable for server and desktop applications.
Dart wasn’t popular and was known as an orphan, but it had a compiler for a Dart VM, also Google had all the control about this language.